scripts/start_bots.sh

```diff
@@ -1,14 +1,42 @@
 #!/bin/bash
 
-# Start the app in the background and save the logs to a file
-nohup uvicorn main:app --port 8000 --host 0.0.0.0 > bot.log 2>&1 &
+read -p "Start demo bot [y/N]? " choice
```

Comment:
I think this question should be in the install script. When you run start_bot it should start the bot.